packages/vue/tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "strict": true,
     "jsx": "preserve",
     "importHelpers": true,
-    "declaration": true,
+    "declaration": false,
```

Comment:
Will this prevent type generation?

Reply:
Our typescript d.ts generation still occurs with our vite `typeScript2` plugin settings.

```
// vite.config.js
    tsconfigOverride: {
        compilerOptions: {
          sourceMap: true,
          declaration: true,
          declarationMap: true,
        },
```

That way we still have typescript definition files being created.

Reply:
Nice! Can we leave a comment here for future readers who may not have context in to `vite` type generation?

Reply:
Sure! I'll add a note